src/types/Health.ts

```diff
@@ -28,10 +44,12 @@ export interface WorkloadStatus {
   availableReplicas: number;
 }
 
+export interface RequestType {
+  [key: string]: { [key: string]: number };
```

Comment:
In general, unless it's a free map like labels, I'd add a comment to indicate what is expected to the keys on this structure:
```
[key: string]: { [key: string]: number }
```

It helps a lot to learn the code, and check if there could be any redundant type.